app/_components/protocol/_protocol.scss

```diff
@@ -0,0 +1,65 @@
+:target + .app-protocol {
+  outline: $govuk-focus-width solid govuk-colour("light-grey");
+}
+
+.app-protocol {
+  .govuk-tag:first-of-type {
+    margin-left: - govuk-spacing(1);
+  }
+
+  .govuk-tag {
+    margin-right: govuk-spacing(1);
+  }
+
+  .govuk-summary-list__value *:first-child {
```

Comment:
Any chance this can have an explicit class?

Reply:
Hmm, doesn’t look like it’s needed anymore, do deleted it.